src/App.tsx

```diff
@@ -1,61 +1,144 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+
+import usersFromServer from './api/users';
+import { todoList, getTodoId, getUser } from './helpers/helpers';
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [hasTitleError, setTitleError] = useState(false);
+  const [hasUserError, setUserError] = useState(false);
+  const [todos, setTodos] = useState(todoList);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value.replace(/[^a-zA-Zа-яА-ЯіІїЇєЄ0-9\s]/g, ''));
+    setTitleError(false);
+  };
```

Comment:
In my opinion, this is not appropriate, since we are writing a name in which we can write dots or commas